src/actions/types.ts

```diff
@@ -5,8 +5,9 @@ export enum Constants {
   START_LOADING = 'START_LOADING',
   STOP_LOADING = 'STOP_LOADING',
   ADD_PAGE = 'ADD_PAGE',
-  ADD_DATA = 'ADD_DATA',
+  ADD_DATA = 'ADD_POSTS',
```

Comment:
Still we have Data as option in Enum, which data we adding?
Rename also enum option not string value only ...